contracts/voting-verifier/src/events.rs

```diff
@@ -119,9 +120,33 @@ pub struct WorkerSetConfirmation {
     pub operators: Operators,
 }
 
+/// If parsing is successful, returns (tx_id, event_index). Otherwise returns ContractError::InvalidMessageID
+fn parse_message_id(
```

Comment:
Why not define such a method for `MessageIdFormat`?

Reply:
this method returns tx_id and event_index. Not every message id format has those two components. They all implement FromStr though.

Reply:
Sure, but it's not like you can ignore those that don't contain those two components in this function. Rust forces you to have all the options in the match block. You'd have to put `unimplemented!()` here anyway. I feel a method on the `MessageIdFormat` is a better indicator that it won't be implemented ever.

Reply:
I just don't like the idea of adding something to `MessageIdFormat` that is not generic. I think if we add formats that can't be parsed to this structure (string, uint), then we need to decide whether this voting verifier will support those formats. If it does, we change this function. If it doesn't, this function errors or panics. I'd rather address that if/when the time comes.